packages/protocol/test/utils.js

```diff
@@ -48,4 +48,14 @@ module.exports = {
     const currentBlock = await provider.getBlock(currentBlockNumber);
     return currentBlock.timestamp;
   },
+  decodeRevertString: (callData) => {
+    // Refer to https://ethereum.stackexchange.com/a/83577
+    // Assuming `callData` is a hex string
+    if (callData.length < 2 + 68 * 2) {
```

Comment:
This seems a little bit different than what was checked in the code from the link above. @bbenligiray you mind explaining what 2 + 68 * 2 means here?

Reply:
I think the idea in the link is that if the call reverted with a string, `callData` will be the signature of the function that reverted (not 100% on this but it doesn't matter) and the revert string. The signature is 4 bytes, the string is minimum 64 bytes. So `callData` should be at least 68 bytes. I'm multiplying by 2 here because we're operating on a hex string (where 2 hex characters is a byte) and the added 2 is the `0x`.

This is outdated though, check out the new implementation.